doc/user/data/disaster_recovery/system_environment_nested.yml

```diff
@@ -0,0 +1,38 @@
+columns:
+  - column: "Component"
+  - column: "Failure Characteristics"
+rows:
+  - "Component": |
+      **Egress gateway**
+
+    "Failure Characteristics": |
+      The egress gateway runs on a single node in a single AZ. Although no data
+      is lost if the egress gateway becomes unavailable, sources and sinks are
+      impacted during this time.
+
+      The component has the following failure characteristics:
+
+      {{< yaml-table data="disaster_recovery/egress_gateway" >}}
```

Comment:
If we want to keep this I'd just call it networking andstate that public sources and sinks may lose connection, as well as single az privatelink connections.

Reply:
Removed